components/music/cards/Artist.tsx

```diff
@@ -77,6 +79,9 @@ const Artist = async ({
         <h2 className="card-title">{artist.name}</h2>
         {isCompact && `${formatNumber(artist.followers.total)} Followers`}
       </div>
+      <div>
+        <PlayContext uri={artist.uri} />
```

Comment:
Move this to the div with the share icon. Possibly change that div's className to `card-actions justify-end`, possibly? Unsure on how to make it look better beyond that.